mittab/libs/data_import/import_rooms.py

```diff
@@ -48,37 +49,49 @@ def import_rooms(fileToImport):
             return room_errors
 
     for i in range(1, num_rooms):
+
+        # headers are
+        # name, room rank
+
         room_name = sh.cell(i, 0).value
         if room_name == '':
-            room_errors.append("Row " + str(i) + ": Empty Room Name")
-            continue
-        try:
-            Room.objects.get(name=room_name)
-            room_errors.append(room_name + ': Duplicate Room Name')
+            room_errors.append("Row " + str(i) + ": Empty room name")
             continue
-        except:
-            pass
 
-        #Load and validate room_rank
+        duplicate = False
+        if Room.objects.filter(name=room_name).first() is not None:  # check for duplicates
+            room_errors.append(room_name + ': Duplicate room name')
+            duplicate = True
+
+        # Load and validate room_rank
         room_rank = sh.cell(i, 1).value
-        room_string = str(room_rank)
         try:
-            room_rank = Decimal(room_rank)
-        except:
-            room_errors.append(room_name + ": Rank not number")
-            continue
-        if len(room_string) > 5 or (room_rank < 10 and len(room_string) > 4):
-            room_errors.append(room_name + ": Rank should have no more than two decimal places")
-            continue
-        if room_rank >= 100 or room_rank < 0:
-            room_errors.append(room_name + ": Rank should be between 0-99.99")
+            # auto-round to two floating point digits
+            room_rank = round(Decimal(room_rank), 2)
+        except TypeError or ValueError:
+            room_errors.append(room_name + ": Rank in file is not a number")
             continue
 
-        #Create the room
-        room = Room(name=room_name, rank=room_rank);
+        # cap room rank at 100
+        if room_rank >= 100:
+            room_rank = 99.99
+            room_errors.append(room_name + ": capped rank to 99.99")
+
+        # floor room rank at 0
+        if room_rank < 0:
+            room_rank = 0
+            room_errors.append(room_name + ": floored rank to 0")
+
+        if not duplicate:  # Create the room
+            room = Room(name=room_name, rank=room_rank)
+
+        else:  # else, update room
```

Comment:
Would prefer not to have this fallback since it would make this import behave differently than others